doc/opencv.bib

```diff
@@ -1016,3 +1016,17 @@ @INPROCEEDINGS{Ke17
   year = {2017},
   organization = {IEEE}
 }
+
+@ARTICLE{gonzalez,
+  title={Digital Image Fundamentals, Digital Imaging Processing},
+  author={Gonzalez, Rafael C and others},
+  year={1987},
+  publisher={Addison Wesley Publishing Company}
+}
+
+@ARTICLE{gruzman,
+  title={Цифровая обработка изображений в информационных системах},
+  author={Грузман, ИС and Киричук, ВС and Косых, ВП and Перетягин, ГИ and Спектор, АА},
+  year={2000},
+  publisher={Изд-во НГТУ Новосибирск}
```

Comment:
Need to check that this is showed correctly.

Reply:
checked, it looks in this way. Is it correctly?
![bib](https://user-images.githubusercontent.com/40189112/43188568-99864598-901e-11e8-932f-38add2d32e2e.png)